Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/FRFSTicketService.hs

```diff
@@ -94,11 +95,13 @@ import qualified Tools.Payment as Payment
 getFrfsRoutes ::
   (Kernel.Prelude.Maybe (Kernel.Types.Id.Id Domain.Types.Person.Person), Kernel.Types.Id.Id Domain.Types.Merchant.Merchant) ->
   Maybe Text ->
+  Maybe Int ->
```

Comment:
Why limit offset is required here ?
We should not add in the UI API.

Reply:
I added this because in Storage I am using query findAllByMerchantOperatingCityAndVehicleType for getting Routes, which uses findAllWithOptionsKV for both case - in normal bus flow getRoutes api and in getRoutes dashboard api. In case of dashboard, limit and offset are expected but here since there is no limit and offset, I kept it in maybe type . Since findAllByMerchantOperatingCityAndVehicleType was expecting it, I introduced them.
If limit and offset are not to be shown on UI side, should I make some other query without findAllWithOptionsKV?